client/web/src/cody/team/TeamMemberList.tsx

```diff
@@ -34,6 +35,18 @@ export interface TeamInvite {
     acceptedAt: string | null
 }
 
+// This tiny function is extracted to make it testable. Same for the "now" parameter.
+export const formatInviteDate = (sentAt: string | null, now?: Date): string => {
+    try {
+        if (sentAt) {
+            return intlFormatDistance(sentAt || '', now ?? new Date())
```

Comment:
Since we already have the `if (sentAt)` the TypeScript compiler knows that the value is truth, and won't give us grief that the type could be `null` right?

So can't we just write this as:

```suggestion
            return intlFormatDistance(sentAt, now ?? new Date())
```